Fails to build on FreeBSD 11.4-RELEASE

unreliablefs seems to use the `EINTEGRITY` identifier which is not available on FreeBSD 11.4-RELEASE. 

Here's a log from the failing build:

<details>
<summary>
Build log
</summary>

Source: http://beefy10.nyi.freebsd.org/data/114i386-default/ed4b8e417ef2/logs/fusefs-unreliablefs-0.1.0_1.log

```
=>> Building sysutils/fusefs-unreliablefs
build started at Tue Jul  6 13:20:04 UTC 2021
port directory: /usr/ports/sysutils/fusefs-unreliablefs
package name: fusefs-unreliablefs-0.1.0_1
building for: FreeBSD 114i386-default-job-12 11.4-RELEASE-p12 FreeBSD 11.4-RELEASE-p12 i386
maintained by: 0mp@FreeBSD.org
Makefile ident: 
Poudriere version: 3.2.8-8-gaf08dbda
Host OSVERSION: 1400008
Jail OSVERSION: 1104001
Job Id: 12

---Begin Environment---
SHELL=/bin/csh
UNAME_p=i386
UNAME_m=i386
OSVERSION=1104001
UNAME_v=FreeBSD 11.4-RELEASE-p12
UNAME_r=11.4-RELEASE-p12
BLOCKSIZE=K
MAIL=/var/mail/root
STATUS=1
HOME=/root
PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin:/root/bin
LOCALBASE=/usr/local
USER=root
LIBEXECPREFIX=/usr/local/libexec/poudriere
POUDRIERE_VERSION=3.2.8-8-gaf08dbda
MASTERMNT=/usr/local/poudriere/data/.m/114i386-default/ref
POUDRIERE_BUILD_TYPE=bulk
PACKAGE_BUILDING=yes
SAVED_TERM=
PWD=/usr/local/poudriere/data/.m/114i386-default/ref/.p/pool
P_PORTS_FEATURES=FLAVORS SELECTED_OPTIONS
MASTERNAME=114i386-default
SCRIPTPREFIX=/usr/local/share/poudriere
OLDPWD=/usr/local/poudriere/data/.m/114i386-default/ref/.p
SCRIPTPATH=/usr/local/share/poudriere/bulk.sh
POUDRIEREPATH=/usr/local/bin/poudriere
---End Environment---

---Begin Poudriere Port Flags/Env---
PORT_FLAGS=
PKGENV=
FLAVOR=
DEPENDS_ARGS=
MAKE_ARGS=
---End Poudriere Port Flags/Env---

---Begin OPTIONS List---
---End OPTIONS List---

--MAINTAINER--
0mp@FreeBSD.org
--End MAINTAINER--

--CONFIGURE_ARGS--

--End CONFIGURE_ARGS--

--CONFIGURE_ENV--
PKG_CONFIG=pkgconf PYTHON="/usr/local/bin/python3.8" XDG_DATA_HOME=/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work  XDG_CONFIG_HOME=/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work  HOME=/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work TMPDIR="/tmp" PATH=/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/.bin:/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin:/root/bin SHELL=/bin/sh CONFIG_SHELL=/bin/sh
--End CONFIGURE_ENV--

--MAKE_ENV--
NINJA_STATUS="[%p %s/%t] " XDG_DATA_HOME=/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work  XDG_CONFIG_HOME=/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work  HOME=/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work TMPDIR="/tmp" PATH=/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/.bin:/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin:/root/bin NO_PIE=yes MK_DEBUG_FILES=no MK_KERNEL_SYMBOLS=no SHELL=/bin/sh NO_LINT=YES DESTDIR=/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/stage PREFIX=/usr/local  LOCALBASE=/usr/local  CC="cc" CFLAGS="-O2 -pipe  -fstack-protector-strong -fno-strict-aliasing -std=c11 "  CPP="cpp" CPPFLAGS=""  LDFLAGS=" -fstack-protector-strong " LIBS=""  CXX="c++" CXXFLAGS="-O2 -pipe -fstack-protector-strong -fno-strict-aliasing  "  MANPREFIX="/usr/local" BSD_INSTALL_PROGRAM="install  -s -m 555"  BSD_INSTALL_LIB="install  -s -m 0644"  BSD_INSTALL_SCRIPT="install  -m 555"  BSD_INSTALL_DATA="install  -m 0644"  BSD_INSTALL_MAN="install  -m 444"
--End MAKE_ENV--

--PLIST_SUB--
CMAKE_BUILD_TYPE="release" PYTHON_INCLUDEDIR=include/python3.8  PYTHON_LIBDIR=lib/python3.8  PYTHON_PLATFORM=freebsd11  PYTHON_SITELIBDIR=lib/python3.8/site-packages  PYTHON_SUFFIX=38  PYTHON_EXT_SUFFIX=.cpython-38  PYTHON_VER=3.8  PYTHON_VERSION=python3.8 PYTHON2="@comment " PYTHON3="" OSREL=11.4 PREFIX=%D LOCALBASE=/usr/local  RESETPREFIX=/usr/local LIB32DIR=lib DOCSDIR="share/doc/unreliablefs"  EXAMPLESDIR="share/examples/unreliablefs"  DATADIR="share/unreliablefs"  WWWDIR="www/unreliablefs"  ETCDIR="etc/unreliablefs"
--End PLIST_SUB--

--SUB_LIST--
PREFIX=/usr/local LOCALBASE=/usr/local  DATADIR=/usr/local/share/unreliablefs DOCSDIR=/usr/local/share/doc/unreliablefs EXAMPLESDIR=/usr/local/share/examples/unreliablefs  WWWDIR=/usr/local/www/unreliablefs ETCDIR=/usr/local/etc/unreliablefs
--End SUB_LIST--

---Begin make.conf---
USE_PACKAGE_DEPENDS=yes
BATCH=yes
WRKDIRPREFIX=/wrkdirs
PORTSDIR=/usr/ports
PACKAGES=/packages
DISTDIR=/distfiles
PACKAGE_BUILDING=yes
PACKAGE_BUILDING_FLAVORS=yes
MACHINE=i386
MACHINE_ARCH=i386
ARCH=${MACHINE_ARCH}
#### /usr/local/etc/poudriere.d/make.conf ####
# XXX: We really need this but cannot use it while 'make checksum' does not
# try the next mirror on checksum failure.  It currently retries the same
# failed mirror and then fails rather then trying another.  It *does*
# try the next if the size is mismatched though.
#MASTER_SITE_FREEBSD=yes
# Build ALLOW_MAKE_JOBS_PACKAGES with 2 jobs
MAKE_JOBS_NUMBER=2
#### /usr/ports/Mk/Scripts/ports_env.sh ####
_CCVERSION_921dbbb2=FreeBSD clang version 10.0.0 (git@github.com:llvm/llvm-project.git llvmorg-10.0.0-0-gd32170dbd5b) Target: i386-unknown-freebsd11.4 Thread model: posix InstalledDir: /usr/bin
_ALTCCVERSION_921dbbb2=none
_CXXINTERNAL_acaad9ca=FreeBSD clang version 10.0.0 (git@github.com:llvm/llvm-project.git llvmorg-10.0.0-0-gd32170dbd5b) Target: i386-unknown-freebsd11.4 Thread model: posix InstalledDir: /usr/bin "/usr/bin/ld" "--eh-frame-hdr" "-dynamic-linker" "/libexec/ld-elf.so.1" "--hash-style=both" "--enable-new-dtags" "-m" "elf_i386_fbsd" "-o" "a.out" "/usr/lib/crt1.o" "/usr/lib/crti.o" "/usr/lib/crtbegin.o" "-L/usr/lib" "/dev/null" "-lc++" "-lm" "-lgcc" "--as-needed" "-lgcc_s" "--no-as-needed" "-lc" "-lgcc" "--as-needed" "-lgcc_s" "--no-as-needed" "/usr/lib/crtend.o" "/usr/lib/crtn.o"
CC_OUTPUT_921dbbb2_58173849=yes
CC_OUTPUT_921dbbb2_9bdba57c=yes
CC_OUTPUT_921dbbb2_6a4fe7f5=yes
CC_OUTPUT_921dbbb2_6bcac02b=yes
CC_OUTPUT_921dbbb2_67d20829=yes
CC_OUTPUT_921dbbb2_bfa62e83=yes
CC_OUTPUT_921dbbb2_f0b4d593=yes
CC_OUTPUT_921dbbb2_308abb44=yes
CC_OUTPUT_921dbbb2_f00456e5=yes
CC_OUTPUT_921dbbb2_65ad290d=yes
CC_OUTPUT_921dbbb2_f2776b26=yes
CC_OUTPUT_921dbbb2_b2657cc3=yes
CC_OUTPUT_921dbbb2_380987f7=yes
CC_OUTPUT_921dbbb2_160933ec=yes
CC_OUTPUT_921dbbb2_fb62803b=yes
_OBJC_CCVERSION_921dbbb2=FreeBSD clang version 10.0.0 (git@github.com:llvm/llvm-project.git llvmorg-10.0.0-0-gd32170dbd5b) Target: i386-unknown-freebsd11.4 Thread model: posix InstalledDir: /usr/bin
_OBJC_ALTCCVERSION_921dbbb2=none
ARCH=i386
OPSYS=FreeBSD
_OSRELEASE=11.4-RELEASE-p12
OSREL=11.4
OSVERSION=1104001
PYTHONBASE=/usr/local
CONFIGURE_MAX_CMD_LEN=524288
HAVE_PORTS_ENV=1
#### Misc Poudriere ####
GID=0
UID=0
DISABLE_MAKE_JOBS=poudriere
---End make.conf---
--Resource limits--
cpu time               (seconds, -t)  unlimited
file size           (512-blocks, -f)  unlimited
data seg size           (kbytes, -d)  524288
stack size              (kbytes, -s)  65536
core file size      (512-blocks, -c)  unlimited
max memory size         (kbytes, -m)  unlimited
locked memory           (kbytes, -l)  unlimited
max user processes              (-u)  89999
open files                      (-n)  1024
virtual mem size        (kbytes, -v)  unlimited
swap limit              (kbytes, -w)  unlimited
socket buffer size       (bytes, -b)  unlimited
pseudo-terminals                (-p)  unlimited
kqueues                         (-k)  unlimited
umtx shared locks               (-o)  unlimited
--End resource limits--
=======================<phase: check-sanity   >============================
===>  License MIT accepted by the user
===========================================================================
=======================<phase: pkg-depends    >============================
===>   fusefs-unreliablefs-0.1.0_1 depends on file: /usr/local/sbin/pkg - not found
===>   Installing existing package /packages/All/pkg-1.16.3.txz
[114i386-default-job-12] Installing pkg-1.16.3...
[114i386-default-job-12] Extracting pkg-1.16.3: .......... done
===>   fusefs-unreliablefs-0.1.0_1 depends on file: /usr/local/sbin/pkg - found
===>   Returning to build of fusefs-unreliablefs-0.1.0_1
===========================================================================
=======================<phase: fetch-depends  >============================
===========================================================================
=======================<phase: fetch          >============================
===>  License MIT accepted by the user
=> ligurio-unreliablefs-0.1.0_GH0.tar.gz doesn't seem to exist in /portdistfiles/.
=> Attempting to fetch https://codeload.github.com/ligurio/unreliablefs/tar.gz/0.1.0?dummy=/ligurio-unreliablefs-0.1.0_GH0.tar.gz
fetch: https://codeload.github.com/ligurio/unreliablefs/tar.gz/0.1.0?dummy=/ligurio-unreliablefs-0.1.0_GH0.tar.gz: size unknown
fetch: https://codeload.github.com/ligurio/unreliablefs/tar.gz/0.1.0?dummy=/ligurio-unreliablefs-0.1.0_GH0.tar.gz: size of remote file is not known
ligurio-unreliablefs-0.1.0_GH0.tar.gz                   32 kB 4378 kBps    00s
=> a3caa832a5a63c69e18b8b66b503f9a460536539.patch doesn't seem to exist in /portdistfiles/.
=> Attempting to fetch https://github.com/ligurio/unreliablefs/commit/a3caa832a5a63c69e18b8b66b503f9a460536539.patch
a3caa832a5a63c69e18b8b66b503f9a460536539.patch        1484  B   11 MBps    00s
===> Fetching all distfiles required by fusefs-unreliablefs-0.1.0_1 for building
===========================================================================
=======================<phase: checksum       >============================
===>  License MIT accepted by the user
===> Fetching all distfiles required by fusefs-unreliablefs-0.1.0_1 for building
=> SHA256 Checksum OK for ligurio-unreliablefs-0.1.0_GH0.tar.gz.
=> SHA256 Checksum OK for a3caa832a5a63c69e18b8b66b503f9a460536539.patch.
===========================================================================
=======================<phase: extract-depends>============================
===========================================================================
=======================<phase: extract        >============================
===>  License MIT accepted by the user
===> Fetching all distfiles required by fusefs-unreliablefs-0.1.0_1 for building
===>  Extracting for fusefs-unreliablefs-0.1.0_1
=> SHA256 Checksum OK for ligurio-unreliablefs-0.1.0_GH0.tar.gz.
=> SHA256 Checksum OK for a3caa832a5a63c69e18b8b66b503f9a460536539.patch.
===========================================================================
=======================<phase: patch-depends  >============================
===========================================================================
=======================<phase: patch          >============================
===>  Patching for fusefs-unreliablefs-0.1.0_1
===>  Applying distribution patches for fusefs-unreliablefs-0.1.0_1
/usr/bin/sed -i.bak 's|freebsd12|freebsd11|'  /wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/tests/util.py  /wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/tests/test_unreliablefs.py
===========================================================================
=======================<phase: build-depends  >============================
===>   fusefs-unreliablefs-0.1.0_1 depends on file: /usr/local/bin/cmake - not found
===>   Installing existing package /packages/All/cmake-3.20.4.txz
[114i386-default-job-12] Installing cmake-3.20.4...
[114i386-default-job-12] `-- Installing curl-7.77.0...
[114i386-default-job-12] |   `-- Installing ca_root_nss-3.63...
[114i386-default-job-12] |   `-- Extracting ca_root_nss-3.63: ........ done
[114i386-default-job-12] |   `-- Installing libnghttp2-1.43.0...
[114i386-default-job-12] |   `-- Extracting libnghttp2-1.43.0: .......... done
[114i386-default-job-12] `-- Extracting curl-7.77.0: .......... done
[114i386-default-job-12] `-- Installing expat-2.4.1...
[114i386-default-job-12] `-- Extracting expat-2.4.1: .......... done
[114i386-default-job-12] `-- Installing jsoncpp-1.9.4...
[114i386-default-job-12] `-- Extracting jsoncpp-1.9.4: .......... done
[114i386-default-job-12] `-- Installing libarchive-3.5.1,1...
[114i386-default-job-12] |   `-- Installing liblz4-1.9.3,1...
[114i386-default-job-12] |   `-- Extracting liblz4-1.9.3,1: .......... done
[114i386-default-job-12] `-- Extracting libarchive-3.5.1,1: .......... done
[114i386-default-job-12] `-- Installing libuv-1.41.0...
[114i386-default-job-12] `-- Extracting libuv-1.41.0: .......... done
[114i386-default-job-12] `-- Installing rhash-1.4.1...
[114i386-default-job-12] |   `-- Installing gettext-runtime-0.21...
[114i386-default-job-12] |   | `-- Installing indexinfo-0.3.1...
[114i386-default-job-12] |   | `-- Extracting indexinfo-0.3.1: .... done
[114i386-default-job-12] |   `-- Extracting gettext-runtime-0.21: .......... done
[114i386-default-job-12] `-- Extracting rhash-1.4.1: .......... done
[114i386-default-job-12] Extracting cmake-3.20.4: .......... done
=====
Message from ca_root_nss-3.63:

--
FreeBSD does not, and can not warrant that the certification authorities
whose certificates are included in this package have in any way been
audited for trustworthiness or RFC 3647 compliance.

Assessment and verification of trust is the complete responsibility of the
system administrator.


This package installs symlinks to support root certificates discovery by
default for software that uses OpenSSL.

This enables SSL Certificate Verification by client software without manual
intervention.

If you prefer to do this manually, replace the following symlinks with
either an empty file or your site-local certificate bundle.

  * /etc/ssl/cert.pem
  * /usr/local/etc/ssl/cert.pem
  * /usr/local/openssl/cert.pem
===>   fusefs-unreliablefs-0.1.0_1 depends on file: /usr/local/bin/cmake - found
===>   Returning to build of fusefs-unreliablefs-0.1.0_1
===>   fusefs-unreliablefs-0.1.0_1 depends on executable: ninja - not found
===>   Installing existing package /packages/All/ninja-1.10.2,2.txz
[114i386-default-job-12] Installing ninja-1.10.2,2...
[114i386-default-job-12] `-- Installing python38-3.8.10...
[114i386-default-job-12] |   `-- Installing libffi-3.3_1...
[114i386-default-job-12] |   `-- Extracting libffi-3.3_1: .......... done
[114i386-default-job-12] |   `-- Installing mpdecimal-2.5.1...
[114i386-default-job-12] |   `-- Extracting mpdecimal-2.5.1: .......... done
[114i386-default-job-12] |   `-- Installing readline-8.1.1...
[114i386-default-job-12] |   `-- Extracting readline-8.1.1: .......... done
[114i386-default-job-12] `-- Extracting python38-3.8.10: .......... done
[114i386-default-job-12] Extracting ninja-1.10.2,2: ........ done
=====
Message from python38-3.8.10:

--
Note that some standard Python modules are provided as separate ports
as they require additional dependencies. They are available as:

py38-gdbm       databases/py-gdbm@py38
py38-sqlite3    databases/py-sqlite3@py38
py38-tkinter    x11-toolkits/py-tkinter@py38
===>   fusefs-unreliablefs-0.1.0_1 depends on executable: ninja - found
===>   Returning to build of fusefs-unreliablefs-0.1.0_1
===>   fusefs-unreliablefs-0.1.0_1 depends on package: pkgconf>=1.3.0_1 - not found
===>   Installing existing package /packages/All/pkgconf-1.7.4,1.txz
[114i386-default-job-12] Installing pkgconf-1.7.4,1...
[114i386-default-job-12] Extracting pkgconf-1.7.4,1: .......... done
===>   fusefs-unreliablefs-0.1.0_1 depends on package: pkgconf>=1.3.0_1 - found
===>   Returning to build of fusefs-unreliablefs-0.1.0_1
===========================================================================
=======================<phase: lib-depends    >============================
===>   fusefs-unreliablefs-0.1.0_1 depends on shared library: libfuse.so - not found
===>   Installing existing package /packages/All/fusefs-libs-2.9.9_2.txz
[114i386-default-job-12] Installing fusefs-libs-2.9.9_2...
[114i386-default-job-12] Extracting fusefs-libs-2.9.9_2: .......... done
=====
Message from fusefs-libs-2.9.9_2:

--
Install the FUSE kernel module (kldload fusefs) to use this port.
===>   fusefs-unreliablefs-0.1.0_1 depends on shared library: libfuse.so - found (/usr/local/lib/libfuse.so)
===>   Returning to build of fusefs-unreliablefs-0.1.0_1
===========================================================================
=======================<phase: configure      >============================
===>  Configuring for fusefs-unreliablefs-0.1.0_1
===>  Performing out-of-source build
/bin/mkdir -p /wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/.build
-- The C compiler identification is Clang 10.0.0
-- The CXX compiler identification is Clang 10.0.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: pkgconf (found version "1.7.4") 
-- Performing Test FUSE_CFLAGS_CHECK
-- Performing Test FUSE_CFLAGS_CHECK - Success
-- Found FUSE: /usr/local/lib/libfuse.so (found suitable version "2.9", minimum required is "2.9") 
-- Looking for fallocate
-- Looking for fallocate - not found
-- Looking for fallocate
-- Looking for fallocate - not found
-- Looking for utimensat
-- Looking for utimensat - found
-- Looking for setxattr
-- Looking for setxattr - not found
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    BOOST_PYTHON_SUFFIX
    CMAKE_COLOR_MAKEFILE
    CMAKE_CXX_FLAGS_RELEASE
    CMAKE_MODULE_LINKER_FLAGS
    CMAKE_SHARED_LINKER_FLAGS
    CMAKE_VERBOSE_MAKEFILE
    Python_ADDITIONAL_VERSIONS
    THREADS_HAVE_PTHREAD_ARG


-- Build files have been written to: /wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/.build
===========================================================================
=======================<phase: build          >============================
===>  Building for fusefs-unreliablefs-0.1.0_1
[ 14% 1/7] /usr/bin/cc -DHAVE_UTIMENSAT -D_FILE_OFFSET_BITS=64 -I/usr/local/include/fuse -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing -std=c11 -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing -std=c11 -D_FILE_OFFSET_BITS=64 -Wall -Wextra -Wno-unused-parameter -MD -MT CMakeFiles/unreliablefs.dir/conf.c.o -MF CMakeFiles/unreliablefs.dir/conf.c.o.d -o CMakeFiles/unreliablefs.dir/conf.c.o -c /wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/conf.c
[ 28% 2/7] /usr/bin/cc -DHAVE_UTIMENSAT -D_FILE_OFFSET_BITS=64 -I/usr/local/include/fuse -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing -std=c11 -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing -std=c11 -D_FILE_OFFSET_BITS=64 -Wall -Wextra -Wno-unused-parameter -MD -MT CMakeFiles/unreliablefs.dir/unreliablefs.c.o -MF CMakeFiles/unreliablefs.dir/unreliablefs.c.o.d -o CMakeFiles/unreliablefs.dir/unreliablefs.c.o -c /wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs.c
[ 42% 3/7] /usr/bin/cc -DHAVE_UTIMENSAT -D_FILE_OFFSET_BITS=64 -I/usr/local/include/fuse -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing -std=c11 -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing -std=c11 -D_FILE_OFFSET_BITS=64 -Wall -Wextra -Wno-unused-parameter -MD -MT CMakeFiles/unreliablefs.dir/unreliablefs_errinj.c.o -MF CMakeFiles/unreliablefs.dir/unreliablefs_errinj.c.o.d -o CMakeFiles/unreliablefs.dir/unreliablefs_errinj.c.o -c /wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errinj.c
FAILED: CMakeFiles/unreliablefs.dir/unreliablefs_errinj.c.o 
/usr/bin/cc -DHAVE_UTIMENSAT -D_FILE_OFFSET_BITS=64 -I/usr/local/include/fuse -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing -std=c11 -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing -std=c11 -D_FILE_OFFSET_BITS=64 -Wall -Wextra -Wno-unused-parameter -MD -MT CMakeFiles/unreliablefs.dir/unreliablefs_errinj.c.o -MF CMakeFiles/unreliablefs.dir/unreliablefs_errinj.c.o.d -o CMakeFiles/unreliablefs.dir/unreliablefs_errinj.c.o -c /wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errinj.c
In file included from /wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errinj.c:16:
In file included from /wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errinj.h:12:
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:9:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:24:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:42:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:69:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:88:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:109:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:127:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:150:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:161:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:173:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:187:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:206:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:223:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:238:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:254:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:272:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:284:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:296:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
/wrkdirs/usr/ports/sysutils/fusefs-unreliablefs/work/unreliablefs-0.1.0/unreliablefs_errno_freebsd.h:315:5: error: use of undeclared identifier 'EINTEGRITY'
    EINTEGRITY,
    ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
ninja: build stopped: subcommand failed.
*** Error code 1

Stop.
make: stopped in /usr/ports/sysutils/fusefs-unreliablefs
=>> Cleaning up wrkdir
===>  Cleaning for fusefs-unreliablefs-0.1.0_1
build of sysutils/fusefs-unreliablefs | fusefs-unreliablefs-0.1.0_1 ended at Tue Jul  6 13:20:12 UTC 2021
build time: 00:00:08
!!! build failure encountered !!!
```

</details>

That's mostly just a heads-up I guess as the 11 branch is almost EOL.

I'll disable building the unreliablefs package for FreeBSD 11 for now.

Cheers!

--- a/unreliablefs_errno_freebsd.h
+++ b/unreliablefs_errno_freebsd.h
@@ -1,12 +1,15 @@
 #ifndef ERRNO_HH
 #define ERRNO_HH
 
+#include <sys/param.h>
 #include <errno.h>
 
 static const int errno_access[] = {
     EACCES,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINVAL,
     EIO,
     ELOOP,
@@ -21,7 +24,9 @@ static const int errno_chmod[] = {
     EACCES,
     EFAULT,
     EFTYPE,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EIO,
     ELOOP,
     ENAMETOOLONG,
@@ -39,7 +44,9 @@ static const int errno_creat[] = {
     EDQUOT,
     EEXIST,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINTR,
     EINVAL,
     EIO,
@@ -66,7 +73,9 @@ static const int errno_ftruncate[] = {
     EBADF,
     EFAULT,
     EFBIG,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINVAL,
     EIO,
     EISDIR,
@@ -85,7 +94,9 @@ static const int errno_link[] = {
     EDQUOT,
     EEXIST,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINVAL,
     EIO,
     ELOOP,
@@ -106,7 +117,9 @@ static const int errno_mkdir[] = {
     EDQUOT,
     EEXIST,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EIO,
     ELOOP,
     EMLINK,
@@ -124,7 +137,9 @@ static const int errno_mknod[] = {
     EDQUOT,
     EEXIST,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINVAL,
     EIO,
     ELOOP,
@@ -147,7 +162,9 @@ static const int errno_opendir[] = {
 static const int errno_readdir[] = {
     EBADF,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINVAL,
     EIO,
 };
@@ -158,7 +175,9 @@ static const int errno_read[] = {
     EBUSY,
     ECONNRESET,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINTR,
     EINVAL,
     EIO,
@@ -170,7 +189,9 @@ static const int errno_read[] = {
 static const int errno_readlink[] = {
     EACCES,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINVAL,
     EIO,
     ELOOP,
@@ -184,7 +205,9 @@ static const int errno_rename[] = {
     ECAPMODE,
     EDQUOT,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINVAL,
     EIO,
     EISDIR,
@@ -203,7 +226,9 @@ static const int errno_rmdir[] = {
     EACCES,
     EBUSY,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINVAL,
     EIO,
     ELOOP,
@@ -220,7 +245,9 @@ static const int errno_symlink[] = {
     EDQUOT,
     EEXIST,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EIO,
     ELOOP,
     ENAMETOOLONG,
@@ -235,7 +262,9 @@ static const int errno_truncate[] = {
     EACCES,
     EFAULT,
     EFBIG,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINVAL,
     EIO,
     EISDIR,
@@ -251,7 +280,9 @@ static const int errno_truncate[] = {
 static const int errno_unlink[] = {
     EACCES,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EIO,
     EISDIR,
     ELOOP,
@@ -269,7 +300,9 @@ static const int errno_write[] = {
     EDQUOT,
     EFAULT,
     EFBIG,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINTR,
     EINVAL,
     EIO,
@@ -281,7 +314,9 @@ static const int errno_write[] = {
 static const int errno_lstat[] = {
     EACCES,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EIO,
     ELOOP,
     ENAMETOOLONG,
@@ -293,7 +328,9 @@ static const int errno_lstat[] = {
 static const int errno_statfs[] = {
     EACCES,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EIO,
     ELOOP,
     ENAMETOOLONG,
@@ -312,7 +349,9 @@ static const int errno_utimensat[] = {
     EACCES,
     EBADF,
     EFAULT,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINVAL,
     EIO,
     ELOOP,
@@ -326,7 +365,9 @@ static const int errno_utimensat[] = {
 
 static const int errno_fsync[] = {
     EBADF,
+#if __FreeBSD__ >= 12
     EINTEGRITY,
+#endif
     EINVAL,
     EIO,
 };

